docs/README.md

```diff
@@ -9,6 +9,7 @@ Afterward, see the below list of topics:
 
 ## Topics for users:
 - [Passing parameters to ServiceInstances and ServiceBindings](./parameters.md)
+- [Walkthrough](walkthrough.md)
```

Comment:
@jpeeler can you move this link to the paragraph above: "... after you install, see our walkthrough document to get started using Service Catalog"